### ISSUE TITLE
No fields showing and Duplicate field options showing after updating

### ISSUE THREAD

**Issue by gab1982** [funkjedi/acf-qtranslate#103]
After updating the plugin (v1.7.22) to work with the latest ACF Pro (5.6.0), I am experiencing some issues, the fields that I had translatable are no longer showing in the admin eg a text field, and when I go into the ACF field options to see what it says, it says 'unknown' for the field name and when I click edit all of the fields are duplicated, I can't even change the field back to just text unless I de-activate the plugin. Please find a screenshot here: https://www.dropbox.com/s/myuh5o2ru3mjqwd/screenshot1.jpg?dl=0 and https://www.dropbox.com/s/hpqk2o7mnvsc6bi/screenshot2.jpg?dl=0

**Comment by paulinhoerry** [comment]
I have same problem

**Comment by gab1982** [comment]
Can anyone help with this, its becoming an issue.

**Comment by gfStudioTesta** [comment]
Same problem here...

**Comment by Bones5** [comment]
This is a major issue, it's due to the new version of ACF.

rolling back to version 5.5.14 of ACF Pro fixes the issue temporarily

**Comment by gfStudioTesta** [comment]
Yes rolling back is a solution. Hope to be not forced to migrate to hatefull WPML 👎

**Comment by paulinhoerry** [comment]
How can I rolling back ACF PRO?

**Comment by Bones5** [comment]
Download version 5.5.14 from here https://www.advancedcustomfields.com/my-account/view-downloads/

delete version 5.6 and upload this zip

**Comment by funkjedi** [comment]
I'll be pushing out a 5.6 compatible version either tonight or tomorrow.

**Comment by gfStudioTesta** [comment]
Any news ?

**Comment by gab1982** [comment]
Yes any news on this?

**Comment by tomanthoni** [comment]
I also have this problem... Client very angry =(

**Comment by cyberfly999** [comment]
Had to roll back to ACF Pro 5.5.14. but can't wait for the fix...

**Comment by gab1982** [comment]
I rolled back too however if a client updates it then will become an issue, hoping for a fix soon...

**Comment by Tusko** [comment]
@gab1982 @paulinhoerry @gfStudioTesta @Bones5 @funkjedi 
Fixed for ACF PRO 5.6.0

https://github.com/Tusko/acf-qtranslate/commit/aefdf4e5645b489be60c57724c10f36ba2319ea7

**Comment by tomanthoni** [comment]
Thanks!

**Comment by gfStudioTesta** [comment]
Ty !

**Comment by gab1982** [comment]
thanks :)

**Comment by cyberfly999** [comment]
Great, thanks :-)

**Comment by jcarignan** [comment]
@Tusko 
So the fix is simply changing the extends to "acf_field" for the 5 file in src/acf_5/fields/ ?

So I guess to keep backward compatibility, a quick (but quite dirty) fix would be something like:

```php
if($acfVersion < 5.6) {
    class acf_qtranslate_acf_5_file_dynamic extends acf_field_file {}
} else {
    class acf_qtranslate_acf_5_file_dynamic extends acf_field {}
}

class acf_qtranslate_acf_5_file extends acf_qtranslate_acf_5_file_dynamic {
 //[...]
}
```

**Comment by Tusko** [comment]
@jcarignan Great idea! 
I did it very fast because too many sites were broken after update to 5.6

**Comment by Tusko** [comment]
@jcarignan added compatibility for older versions

https://github.com/funkjedi/acf-qtranslate/commit/fb47abaf2174d88835c0cab28bfc319879d37d82.patch

**Comment by jnsflint** [comment]
Thanks a lot for taking care of that issue! 

@funkjedi Do you have a ETA of publishing it to the wordpress plugin store?

**Comment by funkjedi** [comment]
Just pushed out update with compatibility fixes. You should see it in your update feed shortly.

**Comment by cedericvh** [comment]
Is it possible I still get duplicate content when saving a wysiwig field ?
I tried everything, also reading these posts.

**Comment by marianagarciaferreira** [comment]
not work with v5.7 and 5.8
